### PR TITLE
Organize archivers

### PR DESCRIPTION
**Pull request by kmk3**
```console
$ git log --reverse --pretty='* %s' master..
* archivers: move archiver-common.inc include to the bottom
* archivers: wrap comments
```

Cc: @glitsj16 (as the author of archiver-common.inc)

---

Note: As mentioned on #3845, please don't squash any commits; use normal merge
or rebase instead.

### CODE REVIEW

**Review by rusty-snake — APPROVED** [review]
"From archiver-common.inc" sound a bit confusing to me, maybe "Included in archiver-common.inc" is more accurate.

**Review by glitsj16 — APPROVED** [review]
I tend to agree with @rusty-snake about the wording 'From ...'. Besides that this is LGTM and can be merged soonish. Thanks for CC'ing me on this.

**Comment by kmk3** [comment]
@rusty-snake left a comment:

> "From archiver-common.inc" sound a bit confusing to me, maybe "Included in
> archiver-common.inc" is more accurate.

@glitsj16 left a comment:

> I tend to agree with @rusty-snake about the wording 'From ...'. Besides that
> this is LGTM and can be merged soonish.

Done.

> Thanks for CC'ing me on this.

No problem.  By the way, it's possible to add a [`CODEOWNERS`][1] file at e.g.:
the root of the repo, so that GitHub automatically adds certain reviewers when
a PR changes a given path.  Example:

```
etc/inc/archiver-common.inc @glitsj16
```

[1]: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners

File: etc/profile-m-z/tar.profile [hunk 1]
```diff
@@ -7,13 +7,17 @@ include tar.local
 # Persistent global definitions
 include globals.local
 
-# Arch Linux (based distributions) need access to /var/lib/pacman. As we drop all capabilities this is automatically read-only.
-noblacklist /var/lib/pacman
-
+# From archiver-common.inc
```
Comment (rusty-snake) [inline etc/profile-m-z/tar.profile:10]: ^

Comment (kmk3) [reply etc/profile-m-z/tar.profile]: My bad, I had only seen it on 7z.profile.  Now it's changed everywhere.

**Comment by rusty-snake** [comment]
merged, thanks.

**Pull request merged**